### ISSUE TITLE
Popovers/ tooltips for LicenseIconography Icons

### ISSUE THREAD

**Issue by akmadian** [creativecommons/vue-vocabulary#21]
**Describe the feature**
I think it could be useful to allow [popovers](https://getbootstrap.com/docs/4.0/components/popovers/) when hovering on the LicenseIconography icons! This could be enabled with a prop. I also think it would be a good idea to allow users to pass the contents of the popover for each icon with some object as a prop for more customization as well.

I think that something like [this](https://wikiki.github.io/elements/tooltip/) would look good, in terms of style.

**Describe any problem solved by the feature**
For explanatory/ educational sections involving icons in the cc chooser, I would like users to be able to hover on an icon and be able to get more information about it.

**Want to take up?**
I could help out, but I'm not sure how much time I'll have to work on it :)

**Comment by akmadian** [comment]
This is a bit of a tack on, but it would be nice to be able to set the icon size with a prop as well. Being able to pass some number of px, em, rem, %, or whatever would be great!

**Comment by dhruvkb** [comment]
@akmadian The icons take the font size of the containing element so putting them in a `<span>` and setting the `font-size` via a CSS directive is the cleanest, sanest approach.

**Comment by Edely** [comment]
Hi @dhruvkb! I am applying to Outreachy. Can I work on this?

**Comment by dhruvkb** [comment]
@Edely, the Popop component has already been developed by @HAMZABoxer98 in #40. Once the PR is merged, you'll be able to add this component to the LicenseIconography icons, but that could take one or two days. Would you like to wait or work on another issue in the meantime?

**Comment by Edely** [comment]
I believe is better to work on another issue, @dhruvkb .

**Comment by dhruvkb** [comment]
@Edely the `Popup` component is now ready for use! Would you like to take this up?

**Comment by Dhruvi16** [comment]
Hey! I would love to take this up.  
Thank you.

**Comment by dhruvkb** [comment]
@Dhruvi16 please go ahead.

**Comment by Dhruvi16** [comment]
I have a silly doubt here. I am able to use the popup on the icons due to 
`{ "props": { "className": "enlarged-text" } }`
I am unable to control the size of the text in the popup. So could you please help with that.
The code :
```
```jsx { "props": { "className": "enlarged-text" } }
<Popup
  to="right"
  color="black">
<LicenseIconography :icon-list="['']"/>
<template #popup size="small">
      That's super cool!
    </template>
</Popup>
```
```
Output -
![Screenshot from 2019-11-16 18-29-57](https://user-images.githubusercontent.com/34674462/68993557-7c82f180-089f-11ea-9b3b-d18936a81056.png)

Also, I am not sure about the message in the popup.
Thank you.

**Comment by Dhruvi16** [comment]
output -
![Screenshot from 2019-11-16 18-29-57](https://user-images.githubusercontent.com/34674462/68993581-b5bb6180-089f-11ea-91b7-875fe0810bd4.png)

**Comment by dhruvkb** [comment]
@Dhruvi16 the enlarged-text prop was just there to show the icons in a larger font size. You may remove the class if it negatively impacts the output.

**Comment by dhruvkb** [comment]
Just so we're clear, we want the user to be able to toggle the popup via a Boolean prop and preferably also be able to pass props to the popup component via `$attrs`.

**Comment by dhruvkb** [comment]
@Dhruvi16 any updates on this?

**Comment by Dhruvi16** [comment]
Hey! I was stuck with university exams. I will get back to this soon.
Sorry for the delay!!

**Comment by Dhruvi16** [comment]
Hey! I have been working on including a popup here and I could do that now. But I am still confused with the way of giving users the ability to toggle the popup. I get that I will use a boolean prop to toggle it but how will the user trigger that?
Thank you
Dhruvi

**Comment by Dhruvi16** [comment]
Is something wrong? I won't be able to proceed with the above doubt. So if anyone can clear that for me that will be awesome.
Thank you.
